### PR TITLE
Migrate docs to GitHub

### PR DESCRIPTION
**Pull request by AayushSaini101**
Migrate the docs of the plugin from the wiki Jenkins, and updated the porm.xml url

### CODE REVIEW

**Review by MarkEWaite — APPROVED** [review]
This looks great!  Thanks for doing it.

I'm happy to merge this pull request "as-is" and deliver a new release of the plugin so that the documentation is taken from the GitHub repository instead of the archived wiki.

Are you interested in splitting the changelog into a CHANGELOG.md so that the README only contains plugin documentation and not plugin history?  That is a preparatory step for the step where the plugin switches to automatically generate its changelog based on pull request descriptions.

I'll need to update the "Contributing to Open Source" document and the Jenkins documentation to include that note, since I don't know if it is mentioned in any of those locations.

**Comment by AayushSaini101** [comment]
> This looks great! Thanks for doing it.
> 
> I'm happy to merge this pull request "as-is" and deliver a new release of the plugin so that the documentation is taken from the GitHub repository instead of the archived wiki.
> 
> Are you interested in splitting the changelog into a CHANGELOG.md so that the README only contains plugin documentation and not plugin history? That is a preparatory step for the step where the plugin switches to automatically generate its changelog based on pull request descriptions.
> 
> I'll need to update the "Contributing to Open Source" document and the Jenkins documentation to include that note, since I don't know if it is mentioned in any of those locations.

Sure sir, I will change it

**Comment by MarkEWaite** [comment]
Also, don't forget to sync your fork each time you start the work on a new pull request.

screencapture-github-MarkEWaite-versionnumber-plugin-2022-09-08-10_35_28-edit

File: CHANGELOG.md [hunk 1]
```diff
@@ -0,0 +1,96 @@
+# Releases of VersionNumber-PLUGIN
```
Comment (MarkEWaite) [inline CHANGELOG.md:1]:
```suggestion
# Releases of Version Number Plugin
```

File: read.md [hunk 2]
```diff
@@ -0,0 +1,69 @@
+# VersionNumber-PLUGIN
```
Comment (MarkEWaite) [inline read.md:1]:
```suggestion
# Version Number Plugin

**Review by MarkEWaite — APPROVED** [review]
Thanks so much!

**Pull request merged**